audit_log/asgi.py

```
@@ -0,0 +1,39 @@
+    Get an ASGI application with audit logging middleware applied.
+    
+    Usage:
+        from audit_log.asgi import get_asgi_application
+        
+        application = get_asgi_application(your_django_asgi_app)
+    """
+    if not ASGI_AVAILABLE:
+        raise ImportError(
+            "ASGI support requires asgiref>=3.2.0. "
+            "Install with: pip install django-audit-log[asgi]"
+        )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)